packages/@atjson/source-ckeditor/test/source-classic-build-editor-test.ts

```diff
@@ -1,20 +1,8 @@
 import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
-import { Annotation } from "@atjson/document";
+import { compareAnnotations } from "@atjson/document";
```

Comment:
Can we remove this in favor of using `sort()`?

Reply:
We're calling this on document.annotations which is an array, not an AnnotationCollection